autorest/client.go

```diff
@@ -31,7 +31,7 @@ import (
 
 const (
 	// DefaultPollingDelay is a reasonable delay between polling requests.
-	DefaultPollingDelay = 60 * time.Second
+	DefaultPollingDelay = 30 * time.Second
```

Comment:
This could end up doubling the number of GET calls during polling which might impact large-scale apps and introduce throttling.  This needs to be investigated before we could merge this as it affects everybody.